src/content/docs/en/developers/developer-quickstart.mdx

```diff
@@ -113,7 +113,7 @@ const config: HardhatUserConfig = {
 To deploy using the Scroll Sepolia Testnet Public RPC, run:
 
 ```bash
-forge create ... --rpc-url=https://sepolia-rpc.scroll.io/ --legacy
```

Comment:
Do we need to wait until Alloy is updated? Can we test this and confirm?

Reply:
I tested this on Scroll Sepolia and it did work already!